app/konnect/dev-portal/applications/disable-app-reg.md

```diff
@@ -3,37 +3,27 @@ title: Disable App Registration for a Service
 no_version: true
 ---
 
-Disabling application registration
-deletes all plugins that were initially enabled for a Service. You cannot manually
-delete a plugin that was automatically enabled by app registration, such as the
-`acl` and `key-auth` or `openid-connect` plugins that
-were automatically enabled in tandem when app registration was enabled.
-When app registration is enabled, you cannot disable
-those authentication plugins either. If you attempt to do so, a modification not
-allowed when application registration message appears. The only way to (indirectly, automatically)
-delete the automatically enabled authentication plugins is to disable app registration.
-Any other plugins that were enabled manually, such as `rate-limiting`, remain enabled.
-
-The main reason for disabling app registration for a Service is when an API
+Disable app registration for a Service is when an API
 no longer requires authentication. If you want to disable Auto Approve at the
```

Comment:
```suggestion
You disable app registration for a Service when an API
no longer requires authentication. If you want to disable Auto Approve at the
```

Or if Disable app registration for a Service is a UI string, mark as bold? (otherwise sentence grammar ... isn't ...)

Reply:
That's an ugly copy-pasta error, nice catch.